// nex-player-challenge/dist/playback-functions.js
"use strict";
//Note that these function all out an animal is an interim only. 
//I want to check whether the functions get called correctly before potentially putting effort into the implementation in vain 
Object.defineProperty(exports, "__esModule", { value: true });
exports.destroy = exports.stop = exports.seek = exports.pause = exports.play = exports.open = exports.setContainer = exports.createPlayer = void 0;
function createPlayer() {
    console.log("Raccoon");
    var player;
    player = document.createElement('video');
    player.autoplay = false;
    player.id = "dashjs-player";
    player.controls = true;
    console.log("Gangsta ");
}
exports.createPlayer = createPlayer;
function setContainer(divId) {
    console.log("Oxen " + divId);
}
exports.setContainer = setContainer;
function open(videoUrl) {
    console.log("Tiger " + videoUrl);
}
exports.open = open;
function play() {
    console.log("Rabbit");
}
exports.play = play;
function pause() {
    console.log("Snake");
}
exports.pause = pause;
function seek(at) {
    console.log("Horse " + at);
}
exports.seek = seek;
function stop() {
    console.log("Goat");
}
exports.stop = stop;
function destroy() {
    console.log("Chimpanzee");
}
exports.destroy = destroy;
